test/integration_test.go

```diff
@@ -87,7 +89,7 @@ func getTest[T require.TestingT](numNode, numTx int) func(t T) {
 		require.NoError(t, err)
 
 		for i := 0; i < numTx; i++ {
-			key := make([]byte, 28) // only 224 bits are available
+			key := make([]byte, 32) // only 224 bits are available
```

Comment:
not sure about that one, probably not very important as the key is hashed anyway. But the comment is wrong ;-)